### ISSUE TITLE
Invalid locale file 'FastForward_chromium_2180_dev\_locales\fr\messages.json': Line: 6, column: 23, Invalid escape sequence.

### ISSUE THREAD

**Issue by azumukupoe** [FastForwardTeam/FastForward#848]
### Body

When I try to install, I get the following error
```
Error
Invalid locale file 'FastForward_chromium_2180_dev\_locales\fr\messages.json': Line: 6, column: 23, Invalid escape sequence.
Could not load manifest.
```

### Version

0.0.0

### What browsers are you seeing the problem on?

Chrome

### What OS are you seeing the problem on?

Windows

**Comment by skoshy** [comment]
I think something is wrong with version 2180, I'm now using 2179 via here and it's working: https://nightly.link/FastForwardTeam/FastForward/actions/runs/3773592105/FastForward_chromium.zip

**Comment by azumukupoe** [comment]
it's https://github.com/FastForwardTeam/FastForward/pull/847, then

**Comment by lostdusty** [comment]
Fixed in #849. Should be working now.
